astropy/cosmology/core.py

```diff
@@ -212,7 +212,7 @@ class FLRW(Cosmology):
     Neff : float, optional
         Effective number of Neutrino species. Default 3.04.
 
-    m_nu : `~astropy.units.Quantity`, optional
+    m_nu : quantity-like, optional
```

Comment:
Agreed. When x-ref on physical types and units is in this can be ``quantity-like [eV]``.
But for now, I'm just doing "Mass of each neutrino species." -> "Mass of each neutrino species in eV."

Reply:
I'd just mimic what is done in the part for T_cmb, to keep the docstring uniform

Reply:
👍    "Mass of each neutrino species in eV." ->  "Mass of each neutrino species in [eV]."